### PR TITLE
Inside Rust: Announcing the Docs.rs Team

### PR DESCRIPTION
**Pull request by QuietMisdreavus**
This is an announcement post for https://github.com/rust-lang/team/pull/182. I set its "publish date" to tomorrow, but that could be optimistic for the team PR's approval. I can change it if need be.

### CODE REVIEW

File: posts/inside-rust/2019-11-14-announcing-the-docsrs-team.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+time went on, those concerns started to diverge more and more, and so did the people who contributed
+to either tool.
+
+The new Docs.rs Team will be responsible for the operations and development for [Docs.rs], leaving
+the Rustdoc Team to be responsible for the Rustdoc tool itself. [@QuietMisdreavus] will be leading
+the new Docs.rs Team, leaving [@GuillaumeGomez] to lead the Rustdoc Team. The Docs.rs Team also
+welcomes [@pietroalbini], who has been helping out a lot with maintaining the infrastructure and
+operations side of Docs.rs.
+
+To get involved with either team, join us [on Discord]! The Rustdoc Team hangs out in `#rustdoc`,
```
Comment (pietroalbini) [inline posts/inside-rust/2019-11-14-announcing-the-docsrs-team.md]: Invite link: https://discord.gg/4yEYPuT

File: posts/inside-rust/2019-11-14-announcing-the-docsrs-team.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+and the Docs.rs Team hangs out in `#docs-rs`.
```
Comment (pietroalbini) [inline posts/inside-rust/2019-11-14-announcing-the-docsrs-team.md]: Invite link: https://discord.gg/2k5vVWn

**Comment by XAMPPRocky** [comment]
@QuietMisdreavus Since this has been sitting for over a week the file now needs to be updated to a new publish date.

**Comment by GuillaumeGomez** [comment]
Well, https://github.com/rust-lang/team/pull/182 has to be merged first.

**Comment by QuietMisdreavus** [comment]
Now that https://github.com/rust-lang/team/pull/182 is merged, i've updated this post with today's date. I've also reworded it to introduce the initial lineup of the team.

**Comment by GuillaumeGomez** [comment]
👍

**Comment by pietroalbini** [comment]
Looks great! @QuietMisdreavus if you update the post date to today I'll merge.

**Comment by QuietMisdreavus** [comment]
@pietroalbini Done! I rebased the branch to get rid of the "rename file" commits, too.

File: posts/inside-rust/2019-12-09-announcing-the-docsrs-team.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Previously, [Docs.rs] has been managed by the Rustdoc Team, as many of the initial concerns of
+Docs.rs were shared by Rustdoc as Docs.rs was being brought into team maintainership. However, as
+time went on, those concerns started to diverge more and more, and so did the people who contributed
+to either tool.
+
+The new Docs.rs Team will be responsible for the operations and development for [Docs.rs], leaving
+the Rustdoc Team to be responsible for the Rustdoc tool itself. [@QuietMisdreavus] will be leading
+the new Docs.rs Team, leaving [@GuillaumeGomez] to lead the Rustdoc Team.
+
+Joining QuietMisdreavus on the Docs.rs Team is GuillaumeGomez, coordinating work between Rustdoc and
```
Comment (GuillaumeGomez) [inline posts/inside-rust/2019-12-09-announcing-the-docsrs-team.md]: You didn't put my username as a link here. :)

Comment (QuietMisdreavus) [reply posts/inside-rust/2019-12-09-announcing-the-docsrs-team.md]: We were both linked in the previous paragraph. 😉

Comment (GuillaumeGomez) [reply posts/inside-rust/2019-12-09-announcing-the-docsrs-team.md]: You forgot to link yourself too! :o

Comment (GuillaumeGomez) [reply posts/inside-rust/2019-12-09-announcing-the-docsrs-team.md]: (I like to link everything hehe)

Comment (QuietMisdreavus) [reply posts/inside-rust/2019-12-09-announcing-the-docsrs-team.md]: I don't like to have a lot of redundant links, especially when you can look one paragraph up to find the same term and see a link there. Personal preference, i guess.

**Pull request merged**